Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
```

Comment:
The rubygems proxy is to make deploys more resilient. The Gemfile is ignored during deploys. It's here only to support `bundle exec rspec`.